src/gwt/acesupport/loader.js

```diff
@@ -198,24 +198,25 @@ oop.inherits(RStudioUndoManager, UndoManager);
 
 function loadEditor(container) {
    var env = {};
+   container.env = env;
```

Comment:
The noise below is just indentation changes -- the major change we make is ensure that the Ace editor is attached to the DOM element (so we can easily access it ourselves if needed). Note that Ace will automatically clean up these references itself when it's destroyed so we shouldn't have to worry about leaks.